src/testdrive/cases.py

```diff
@@ -18,7 +18,12 @@ def timing(cases):
     tv_start = tv_end = duration_last = None
     for case in cases:
         if 'timestamp' in case:
-            tv_case = timevalue(case['timestamp'])
+            try:
+                tv_case = timevalue(case['timestamp'])
+            except TypeError:
```

Comment:
Do we care about other exceptions?

Reply:
Expectation is that `case['timestamp']` contains either an absolute timestamp (ISO 8601 date-time) string or a decimal number. We care that `timevalue` returns a value for absolute timestamps only. If `case['timestamp']` is a string which is not ISO 8601, then we would rather that blew up.